UndertaleModLib/Models/UndertaleTexturePageItem.cs

```diff
@@ -76,5 +86,99 @@ public override string ToString()
         {
             return "(" + GetType().Name + ")";
         }
+
+        public void ExportAsPNG(UndertaleData data, string FullPath, string imageName, bool newTarget)
+        {
+            int exportWidth = BoundingWidth; // sprite.Width
+            int exportHeight = BoundingHeight; // sprite.Height
+            if (newTarget)
+                renderTarget = new WriteableBitmap(exportWidth, exportHeight, 96, 96, PixelFormats.Pbgra32, null);
+
+            // Clear the render target.
+            if (pixelBuffer == null)
+            {
+                foreach (var texPageItem in data.TexturePageItems)
+                {
+                    if ((int)texPageItem.BoundingWidth > maxSpriteWidth)
+                        maxSpriteWidth = texPageItem.BoundingWidth;
+                    if ((int)texPageItem.BoundingHeight > maxSpriteHeight)
+                        maxSpriteHeight = texPageItem.BoundingHeight;
+                }
+                pixelBuffer = new UInt32[maxSpriteWidth * maxSpriteHeight]; // Allocate the memory buffer.
+            }
+
+            for (int x = 0; x < pixelBuffer.Length; x++)
+                pixelBuffer[x] = (UInt32)0U;
+
+            System.Windows.Int32Rect rect = default;
```

Comment:
My Visual Studio complains that `= default` is a part of C# 7.1 and the project is configured for C# 7.0

Reply:
Will fix.